common-util/src/test/java/org/sunbird/services/sso/impl/KeyCloakUnitTest.java

```diff
@@ -0,0 +1,96 @@
+package org.sunbird.services.sso.impl;
+
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.security.PublicKey;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.common.models.util.KeyCloakConnectionProvider;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+// ** @author kirti. Junit test cases *//*
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  HttpClientBuilder.class,
+  CloseableHttpClient.class,
+  HttpGet.class,
+  CloseableHttpResponse.class,
+  HttpResponse.class,
+  HttpEntity.class,
+  EntityUtils.class,
+  JsonObject.class,
+  JsonParser.class
+})
+public class KeyCloakUnitTest {
+
+  public static final String FAIL_TEST = "fail-check";
+  private static final HttpClientBuilder httpClientBuilder =
+      PowerMockito.mock(HttpClientBuilder.class);
+  private static CloseableHttpClient client = null;
+  private static CloseableHttpResponse response;
+  private static HttpEntity httpEntity;
+  private final JsonParser jsonParser = PowerMockito.mock(JsonParser.class);
+
+  @Before
+  public void setUp() throws Exception {
+
+    client = PowerMockito.mock(CloseableHttpClient.class);
+    PowerMockito.mockStatic(HttpClientBuilder.class);
+    when(HttpClientBuilder.create()).thenReturn(httpClientBuilder);
+    when(httpClientBuilder.build()).thenReturn(client);
+    httpEntity = PowerMockito.mock(HttpEntity.class);
+    PowerMockito.mockStatic(EntityUtils.class);
+    PowerMockito.whenNew(JsonParser.class).withNoArguments().thenReturn(jsonParser);
+  }
+
+  @Test
+  public void getPublicKeyFromKeyCloakTest() throws Exception {
```

Comment:
Rename all tests in KeyCloakRsaKeyFetcherTest in below format.
e.g. getPublicKeyFromKeyCloakTest -> testGetPublicKeyFromKeyCloakSuccess